packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -507,6 +507,7 @@ describe('TokenListController', () => {
     const messenger = getRestrictedMessenger(controllerMessenger);
     const controller = new TokenListController({
       chainId: NetworksChainId.mainnet,
+      selectedAddress: '0x1',
```

Comment:
I see that `selectedAddress` was added to all of the places where TokenListController is instantiated, but the functionality added isn't being tested. Can we add tests? Keep in mind that we don't need a real PreferencesController to test the new callback.